### ISSUE TITLE
On iPad, if a port is dragged, the whole node is moved

### ISSUE THREAD

**Issue by ctzurcanu** [clientIO/joint#926]
plus it leaves the link end anchor trailing behind.
Example:
https://resources.jointjs.com/demos/devs 
delete one of the links and try to re-link the ports

**Comment by kumilingus** [comment]
The issue is caused by `[Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See https://www.chromestatus.com/features/5093566007214080`.

**Comment by kumilingus** [comment]
Temporary workaround.
```javascript
joint.dia.ElementView.prototype.pointerdown = function(evt, x, y) {
       var data = this.eventData(evt);
        if (data.stopPropagation) return;
        joint.dia.CellView.prototype.pointerdown.apply(this, arguments);
        this.notify('element:pointerdown', evt, x, y);
        this.dragStart(evt, x, y);
    };
```

**Comment by ctzurcanu** [comment]
this workaround worked for me. thanks. I am not sure if I should close the issue. Your decision.

**Comment by kumilingus** [comment]
We'll leave this open until this gets fixed in JointJS. Thank you.

**Comment by kumilingus** [comment]
Fixed.

**Comment by kumilingus** [comment]
Fix in `v2.2.0`.
